### ISSUE TITLE
There was a problem downloading the bundle. Please try again in a moment.

### ISSUE THREAD

**Issue by ajs256** [adafruit/circup#134]
- Platform/operating system: macos 12.1

- Python version (run `python -version` or `python3 -version`):  3.9.9

- Error message you are receiving, including any Python exception traces:  
```
ajs256@greeble ~ $ circup install adafruit_macropad
Found device at /Volumes/CIRCUITPY, running CircuitPython 7.0.0.
Downloading latest version for adafruit/Adafruit_CircuitPython_Bundle.

  [####################################]  100%          
There was a problem downloading the bundle. Please try again in a moment.
```

- List the steps to reproduce the problem below (if possible attach code or commands
  to run): Attempt to download any library

**Comment by FoamyGuy** [comment]
This should be resolved with #132. Make sure that you have circup versions `1.0.1` https://github.com/adafruit/circup/releases/tag/1.0.1 it was released just the other day so you may need to pip update if you haven't yet.

**Comment by ajs256** [comment]
That worked, thanks!
